cli/user/src/test/java/io/pravega/cli/user/scope/ScopeCommandsTest.java

```diff
@@ -59,6 +59,8 @@ public void testCreateScope() {
         String commandResult = TestUtils.executeCommand("scope create " + scope, cliConfig());
         Assert.assertTrue(commandResult.contains("created successfully"));
         Assert.assertNotNull(ScopeCommand.Create.descriptor());
+
+        String cleanUp = TestUtils.executeCommand("scope delete " + scope, cliConfig());
```

Comment:
is this a left over ?

Reply:
No, I added it on purpose to remove the created scope in its method as a clean up.